apps/remix-ide/src/app/panels/layout.ts

```diff
@@ -64,6 +66,14 @@ export class Layout extends Plugin {
           break
       }
     })
+    this.on('sidePanel', 'focusChanged', async (name) => {
+      const current = await this.call('sidePanel', 'currentFocus')
+      if (this.maximised[current]) {
+        this.event.emit('maximisesidepanel')
```

Comment:
Not sure if event name should camelcase

Reply:
the other related event 'minimizesidepanel' was already using this case... I kept the same logic.